bin/helm

```diff
@@ -2,7 +2,11 @@
 
 set -eu
 
-helmversion=v3.8.0
```

Comment:
Probably for a separate PR, but since Helm is in the devcontainer, we can probably get rid of this script.

Reply:
I agree it would be good to get rid of these scripts but trying not to pull the rug out from folks who use other editors.

Reply:
i'm mildly in favor of keeping them because it seems kind of like a shame to cut out anyone not using VS Code entirely (do any other editors have devcontainer support?)

Reply:
https://github.com/devcontainers/spec is editor-agnostic. https://github.com/jamestthompson3/nvim-remote-containers, for instance, is a neovim plugin that uses devcontainers.

i haven't tested any of this, but it should in theory be possible.

Also, removing these tests doesn't actually prevent folks from working with the repo, they're just on the hook for installing tools themselves.

Generally, all of this stuff is providing a golden path. Our golden path uses devcontainers. I think, ultimately, we should remove cruft that duplicates what we provide via the devcontainer. That can be incremental, though.